internal/storage/bucket_handle.go

```diff
@@ -512,13 +512,8 @@ func (b *bucketHandle) GetFolder(ctx context.Context, folderName string) (*gcs.F
 		return nil, err
 	}
 
-	folder := gcs.Folder{
-		Name:           clientFolder.Name,
-		Metageneration: clientFolder.Metageneration,
-		UpdateTime:     clientFolder.GetUpdateTime().AsTime(),
-	}
-
-	return &folder, err
+	folder := gcs.ControlFolderAttrsToGCSFolder(b.bucketName, clientFolder)
```

Comment:
ControlFolderAttrsToGCSFolder can be renamed to ConvertToGCSFolderFromGetFolderResponse, method name makes more sense as this method main purpose is conversion and also should avoid short forms and acronyms in method names like Attrs.

Reply:
Done.